src/picframe.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
Nice addition. Please move to the root directory, not `src`. `src` should only contain Rust code. You could also add a `utils` directory under the root, or so.